generators/element/index.js

```diff
@@ -376,6 +348,15 @@ module.exports = class extends Generator {
         console.log(util.inspect(this.props, { showHidden: false, depth: 4 }));
         console.log("-------------------------------------------//\n");
       }
+
+      // Generate test boilerplate using a subgenerator.
+      this.composeWith(require.resolve('../test'), {
```

Comment:
In this file we're moving the "test" specific bits of the generator to their own sub-generator so they can be called as a standalone generator.